conf/evolutions/collins/14.sql

```diff
@@ -0,0 +1,11 @@
+# --- Add GPU information to asset_meta
+
+# --- !Ups
+
+INSERT INTO asset_meta (name, priority, label, description) VALUES ('GPU_PRODUCT', -1, 'GPU Product', 'GPU product (description of GPU)');
```

Comment:
Yea, that is fair. If its not a useful field, lets ignore it. I was thinking it could contain some more useful information, but apparently not! :dancer: